front/pages/w/[wId]/developers/api-keys.tsx

```diff
@@ -58,13 +65,26 @@ export function APIKeys({
   groups: GroupType[];
 }) {
   const { mutate } = useSWRConfig();
-  const globalGroup = groups.find((group) => group.kind === "global");
+  const nonGlobalGroups = groups.filter((g) => g.kind != "global");
```

Comment:
This should also let you pick the global workspace. Let's make sure to be consistent regarding names throughout the App.